### ISSUE TITLE
API for Dubbing Studio

### ISSUE THREAD

**Issue by xhsonny** [elevenlabs/elevenlabs-python#292]
Hi, thanks for the great work. Dubbing Studio is amazing! I have been using its web [interface](https://elevenlabs.io/docs/dubbing/studio) to edit especially in the speaker cards to change sentences/words. [API](https://elevenlabs.io/docs/api-reference/create-dub) supports creating and downloading dub. I wonder when the functionaries of Dubbing Studio will be supported by the API. Thanks!

**Comment by lharries** [comment]
Hi @xhsonny, which functionality are you looking for specifically?

**Comment by xhsonny** [comment]
@lharries Thanks for the reply! I am hoping to edit words/sentences in the speaker cards then download the dubbed video with those edited words/sentences through API. I can do this through the web interface now.

**Comment by lharries** [comment]
Thanks - are you looking to do the edits via the API too?

**Comment by xhsonny** [comment]
Yup. Create a project in Dubbing Studio -> edit in speaker cards -> download the edited version, all through API.

**Comment by gu-ma** [comment]
Hey, I'm looking at something similar. Like assigning default speakers for dubbing (not doing a voice clone automatically), and Create Dubbing Studio via the api as well... The api is a bit limited for dubbing atm. Is it possible to do that somehow already? Or is it planned?

**Comment by AngeloGiacco** [comment]
hey , we now have a dubbing api, but i will add it as a feature request to make it more feature complete. as this isnt an issue with our SDK per-se, i'm closing here for now. thanks as always for the feedback!

**Comment by gu-ma** [comment]
Hey, sorry for reopening that issue. There are some changes on the API, but it's all for Enterprise users:

<img width="216" alt="Image" src="https://github.com/user-attachments/assets/c602520c-d63b-494a-8c0e-d9b5bb72cc1c" />

There is no way to update the speaker id for "normal" users?

**Comment by PaulAsjes** [comment]
@gu-ma that's an enterprise only feature for now.
